### ISSUE TITLE
About GHM loss using BCE or BCEWithLogit

### ISSUE THREAD

**Issue by lancesye** [Ariel97-star/FFNet-CoronaryArtery-Segmentation#2]
In your article, the EQ.8 definte the GHM loss.  p = sigmoid(x) LGHM = ∑ LBCE(pi; t)/ GD(gi). 
But in your code: F.binary_cross_entropy_with_logits(pred, target, weights, reduction='sum') / tot. BCEWithLogits will add a sigmoid to pred. Infact, pred had activated by sigmoid. So why use the BCEWithLogits instead of BCE. This does not match the formula.
